util/src/main/scala/org/ensime/util/file.scala

```diff
@@ -6,11 +6,10 @@ import java.io.{ File => JFile, _ }
 import java.nio.charset.Charset
 import java.util.regex.Pattern
 import java.nio.file.Files
+import scala.io.Source
```

Comment:
can you please do this using the NIO library instead of `scala.io.Source`? The latter is **very** inefficient.